vck-openid/src/commonMain/kotlin/at/asitplus/wallet/lib/openid/PresentationFactory.kt

```diff
@@ -200,15 +200,15 @@ internal class PresentationFactory(
             addX5c = false
         ).getOrElse {
             Napier.w("Could not sign id_token", it)
-            throw OAuth2Exception(Errors.USER_CANCELLED, it)
+            throw AccessDenied("Could not sign id_token", it)
```

Comment:
and this one too?

Reply:
Yes, this has now been clarified in OpenID4VP, and we're using it here